docs/05-technical-reference/00-configuration-parameters/obsv-31-configpara-kiali.md

```diff
@@ -4,10 +4,6 @@ title: Kiali Chart
 
 To configure the Kiali chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
 
```

Comment:
Don't we want to convert this old TIP (lines 7-10) into this one instead of scraping it altogether? 🤔 
I thought we agreed on using this:

```suggestion

> **TIP:** See how to [change Kyma settings](https://github.com/kyma-project/kyma/blob/kyma-2.0-docu/docs/04-operation-guides/operations/03-change-kyma-config-values.md).
```

Reply:
If there's such an agreement, all configuration parameters docs should have this, is that correct? 🤔

Reply:
I believe so.